.eslintrc.cjs

```diff
@@ -625,13 +626,110 @@ module.exports = {
     'wrap-regex': 'off',
     yoda: 'off',
 
-    'unicorn/explicit-length-check': 'error',
-    'unicorn/no-array-for-each': 'error',
-    'unicorn/no-negated-condition': 'error',
-    'unicorn/numeric-separators-style': 'error',
-    'unicorn/prefer-default-parameters': 'error',
-    'unicorn/prefer-includes': 'error',
-    'unicorn/template-indent': 'error',
```

Comment:
these are all active in the recommended config, so we don't have to list them out explicitly